src/Commands/InstallHooks.php

```diff
@@ -32,11 +34,12 @@ public function handle()
         $srcPath = __DIR__ . '/../Hooks/';
         $destPath = base_path() . '/.git/hooks/';
 
-        $files = ['pre-commit', 'pre-push'];
+        $finder = new Finder();
```

Comment:
would you mind type hinting this dependency into the constructor so that the container resolves it? Thanks!

Reply:
@mr-feek updated as you comment.